src/trace_processor/trace_processor_shell.cc

```diff
@@ -1718,10 +1723,13 @@ base::Status MaybeUpdateSqlModules(const CommandLineOptions& options) {
     }
   }
 
-  if (!options.sql_module_path.empty()) {
-    auto status = IncludeSqlModule(options.sql_module_path, false);
-    if (!status.ok())
-      return base::ErrStatus("Couldn't add SQL module: %s", status.c_message());
+  if (!options.sql_module_paths.empty()) {
+    for (const auto& add_sql_module_path : options.sql_module_paths) {
+      auto status = IncludeSqlModule(add_sql_module_path, false);
```

Comment:
While you're here please refactor `IncludeSqlModule` (specifically Module -> Package).